source/Calamari/Commands/FindPackageCommand.cs

```diff
@@ -40,9 +40,8 @@ public override int Execute(string[] commandLineArguments)
             Guard.NotNullOrWhiteSpace(packageId, "No package ID was specified. Please pass --packageId YourPackage");
             Guard.NotNullOrWhiteSpace(rawPackageVersion, "No package version was specified. Please pass --packageVersion 1.0.0.0");
             Guard.NotNullOrWhiteSpace(packageHash, "No package hash was specified. Please pass --packageHash YourPackageHash");
-            
-            var fileSystem = CalamariPhysicalFileSystem.GetPhysicalFileSystem();
-            var extractor = new GenericPackageExtractorFactory().createJavaGenericPackageExtractor(fileSystem);
+
+            var extractor = new GenericPackageExtractorFactory().createJavaGenericPackageExtractor(null);
```

Comment:
We dont actually _do_ any extraction in a FindPackage invocation, so its not important to pass a JarTool class in (which expects certain variables to be present)